public/video-ui/src/actions/UploadActions/getUploads.js

```diff
@@ -10,8 +10,6 @@ function runningUploads(uploads) {
 }
 
 function errorGettingUploads(error) {
-  Logger.error(error);
```

Comment:
Sounds sensible. Would be useful to do this for other error reporting as well later?